### DISCUSSION TITLE
DatasetAdapter nome dos campos

### DISCUSSION THREAD

**Discussion by MiltonSonoda** [viniciussanchez/RESTRequest4Delphi#87] [Q&A] [answered]
Vinícius, como evito nome do campo, por exemplo "IdProduto" ser transformado em "Id_Produto". Vi que se trata de uma propriedade to Dataset-serialize. Mas como defino isso direto no ResRequest4Delphi?

**Comment by viniciussanchez** [comment, marked as answer]
No DPR do seu projeto, você pode colocar isso:
```pascal
TDataSetSerializeConfig.GetInstance.CaseNameDefinition := cndLower;
```
Ai onde está o `cndLower` você escolhe a forma como quer trabalhar...

**Comment by MiltonSonoda** [comment]
Muuuito Obrigado!

Abs,


Em qui., 17 de mar. de 2022 às 15:54, Vinicius Sanchez <
***@***.***> escreveu:
